ludwig/utils/backward_compatibility.py

```diff
@@ -836,3 +840,13 @@ def _is_old_missing_value_strategy(feature_config: FeatureConfigDict):
     if not missing_value_strategy or missing_value_strategy not in ("backfill", "pad"):
         return False
     return True
+
+
+def _is_image_feature(feature_config: FeatureConfigDict):
+    preproc = feature_config.get(PREPROCESSING, {})
+    return HEIGHT in preproc and WIDTH in preproc
+
+
+def _update_old_image_preprocessing(feature_config: FeatureConfigDict):
+    standardize_image = feature_config.get(PREPROCESSING, {}).get("standardize_image", None)
+    feature_config[PREPROCESSING].update({"standardize_image": standardize_image})
```

Comment:
This line will fail if the preprocessing section is missing, as we assume may be the case in the line above. To make it more robust:

```
preprocessing = feature_config.get(PREPROCESSING)
if not preprocessing:
    return
preprocessing["standardize_image"] = preprocessing.get("standardize_image")
```